pkgs/top-level/aliases.nix

```diff
@@ -283,6 +283,7 @@ mapAliases ({
   dbus_libs = throw "'dbus_libs' has been renamed to/replaced by 'dbus'"; # Converted to throw 2022-02-22
   dbus_tools = throw "'dbus_tools' has been renamed to/replaced by 'dbus.out'"; # Converted to throw 2022-02-22
   dbvisualizer = throw "dbvisualizer has been removed from nixpkgs, as it's unmaintained"; # Added 2020-09-20
+  dd-agent = throw "dd-agent has been removed in favor of the newer datadog-agent"; # Added 2022-04-26
```

Comment:
Similar to this, you'll want something in nixos/modules/... using mkRemovedOptionModule to give a better error message when configs are using the old services.dd-agent options. Either put it in the datadog-agent module .nix file, or in rename.nix with other deprecated modules (I don't know which of the two is more idiomatic, I suspect either is fine).

Reply:
I've added the `mkRemovedOptionModule` and rebased in https://github.com/NixOS/nixpkgs/pull/187857.